public/server/db/dp.json

```
@@ -0,0 +1,4 @@
+{
+  "definition": "Display Picture"
```

There seems to be a missig comma at the end of the `definition`
```suggestion
  "definition": "Display Picture",
```